flare_portal/experiments/models/modules.py

```diff
@@ -611,3 +611,37 @@ def get_module_config(self) -> constants.ModuleConfigType:
 
     def __str__(self) -> str:
         return "Task instructions - " + super().__str__()
+
+
+def construct_awareness_question(first_keyword, second_keyword):
+    return (
+        "Did you happen to notice whether you heard the "
+        + f"{first_keyword} after seeing a certain {second_keyword}?"
+    )
```

Comment:
Suggestion: Put this code inside of `get_module_description` and call `self.get_module_description()` to get the awareness question in `get_module_config`